cyhy/db/ticket_manager.py

```diff
@@ -244,8 +250,8 @@ def open_ticket(self, vuln, reason):
         new_ticket.save()
         self.__mark_seen(new_ticket)
 
-        # Create notifications for Highs (3) or Criticals (4)
-        if new_ticket["details"]["severity"] > 2:
+        # Create notifications for Highs (3) or Criticals (4) or if KEV is true
```

Comment:
Just something to break up the chain of `or`s since the first two are part of one logical check.

```suggestion
        # Create notifications for Highs (3) or Criticals (4), or if KEV is true
```